src/topology/multicodec-topology.js

```diff
@@ -55,43 +55,44 @@ class MulticodecTopology extends Topology {
 
   /**
    * Update topology.
-   * @param {Array<PeerInfo>} peerInfoIterable
+   * @param {Array<PeerData>} peerDataIterable
```

Comment:
Where is `PeerData` declared? We need a type def, or just document the literal object here.
```suggestion
   * @param {Array<{id: PeerId, multiaddrs: Array<Multiaddr>}>} peerDataIterable
```